documentation/book/ref-kafka-bridge-consumer-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-consumer-configuration-{context}']
+= Kafka Bridge Consumer configuration
+
+Kafka Bridge consumer is configured using the properties in `KafkaBridge.spec.consumer`.
+This property contains the Kafka Bridge consumer configuration options as keys.
+The values can be one of the following JSON types:
+
+* String
+* Number
+* Boolean
+
+You can specify and configure the options listed in the {ApacheKafkaConsumerConfig}.
```

Comment:
All options?

Reply:
I believe yes. @ppatierno ?

Reply:
I would say yes, but avoid to specify "all" for now :-)